src/docker-images/RepairManagerAgent/Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM python:3.7
+FROM ubuntu:18.04
 MAINTAINER Deborah Sandoval <Deborah.Sandoval@microsoft.com>
 
 RUN apt-get update && apt-get --no-install-recommends install -y \
+    python3.7 \
+    python3-pip \
     systemd-sysv
 
-RUN pip3 install wheel setuptools requests pyyaml
+RUN pip3 install wheel setuptools requests
+RUN pip3 install pyyaml
```

Comment:
Why not merge with above line?

Reply:
pyyaml has a dependency on setuptools and error would occur during docker build

Reply:
ok, you should add a comment

Reply:
![image](https://user-images.githubusercontent.com/8230841/80782931-e7ae4080-8b2c-11ea-934c-ebf6d6bc158e.png)